drift-client/README.md

```diff
@@ -108,16 +122,19 @@ is used in the example above, provides `DriftNettyMethodInvokerFactory`, and Apa
 `ApacheThriftMethodInvokerFactory`.  Each transport requires sightly different configuration, 
 so each transport provides a different configuration class.
 
-# Guice Support
+## Guice Support
 
 Drift includes optional support for binding clients into Guice.
 
 ## Client Binding
```

Comment:
Should have three `#`

Reply:
Actually, this heading can be removed since there is only a single section for "Guice Support"